AUTHORS

```diff
@@ -0,0 +1,3 @@
+Daniel Huppmann <huppmann@iiasa.ac.at> (@danielhuppmann, IIASA)
+Franceso Lovat <lovat@iiasa.ac.at> (@francescolovat, IIASA)
+Paul Natsuo Kishimoto <kishimot@iiasa.ac.at> (@khaeru, IIASA)
```

Comment:
I would feel it's only fair if @khaeru is the first name on the list of authors...

```suggestion
Paul Natsuo Kishimoto <kishimot@iiasa.ac.at> (@khaeru, IIASA)
Daniel Huppmann <huppmann@iiasa.ac.at> (@danielhuppmann, IIASA)
Franceso Lovat <lovat@iiasa.ac.at> (@francescolovat, IIASA)
```

Reply:
Thanks!